Buildings/Fluid/AirFilters/BaseClasses/FlowCoefficientCorrection.mo

```diff
@@ -1,26 +1,28 @@
 within Buildings.Fluid.AirFilters.BaseClasses;
 model FlowCoefficientCorrection
   "Component that calculates the flow coefficient correction factor"
-  parameter Real b
-    "Resistance coefficient";
+  parameter Buildings.Fluid.AirFilters.BaseClasses.Data.Generic per
```

Comment:
Do we need to instantiate the `per` parameter in the base class? I think it may be enough to have the `per` been instantiated only in the top level class, i.e. `Buildings.Fluid.AirFilters.Empirical`.

Reply:
Change it as suggested